binding.cc

```diff
@@ -942,6 +952,17 @@ namespace zmq {
           return NanThrowTypeError("Option must be a positive integer");
       }
     }
+
+    if (args.Length() > 1) {
+      if (!args[1]->IsUndefined()) {
```

Comment:
I wanted to keep the same logic as already was there (for args[0]). I think that it would be better to make all those validations in the "js" file and always call the CPP with 2 arguments, so the cpp would not need to validate again.